eel/eel-editable-label.c

```diff
@@ -1039,7 +1038,6 @@ eel_editable_label_ensure_layout (EelEditableLabel *label,
                 longest_paragraph = width;
 
                 wrap_width = get_label_wrap_width (label);
-                scale = gtk_widget_get_scale_factor (widget);
```

Comment:
this has nothing to do with this changeset, but looks unused indeed.

Reply:
If we need to I can split that out as another PR, but as the build warning kept popping up I figured best to catch it now, as there's no possible harm in removing a variable that is in fact unused

Reply:
I think it can stay; I usually don't like mixing unrelated stuff in commits & PRs, but that one looks really iniquitous.

> there's no possible harm in removing a variable that is in fact unused

Although it's most likely true here, that's not necessarily always the case. The biggest risk is actually removing the function call that could have side effects you didn't expect.  This said, doesn't matter here.